servant-docs/src/Servant/Docs/Internal.hs

```diff
@@ -367,6 +369,37 @@ docsWith opts intros (ExtraInfo endpoints) p =
 docsWithIntros :: HasDocs layout => [DocIntro] -> Proxy layout -> API
 docsWithIntros intros = docsWith defaultDocOptions intros mempty
 
+-- | Prettify generated JSON documentation.
```

Comment:
Could this go in a separate module (e.g. `Docs/Internal/Pretty.hs`), and rexported? I tend to think `Internal.hs` is already getting too big, and this is pretty self-contained.